src/main/java/io/vertx/redis/RedisClient.java

```diff
@@ -20,6 +21,15 @@ static RedisClient create(Vertx vertx) {
     return new RedisClientImpl(vertx, new RedisOptions());
   }
 
+  /**
+   * For type safety please use {@link #create(Vertx, RedisOptions)}.
+   */
```

Comment:
Missing @deprecated in the javadoc:

@Deprecated For type safety please use {@link #create(Vertx, RedisOptions)}.